test/Test.agda

```diff
@@ -1,40 +1,55 @@
-
 module _ where
 
 open import Agda.Builtin.List
 open import Agda.Builtin.Nat
+open import Agda.Builtin.Bool
+open import Agda.Builtin.Float
+open import Agda.Builtin.Word
+open import Agda.Builtin.Char
 open import Agda.Builtin.Equality
 
 variable
   a b : Set
 
+-- TODO: move to Prelude.hs?
```

Comment:
The test suite is set up to handle multiple files so I think it makes sense to split up this file into smaller pieces. (Just remember to make all modules reachable from `AllTests.agda`).